windows/src/engine/kmtip/kmkey.cpp

```diff
@@ -265,24 +279,34 @@ HRESULT WINAPI CKeymanEditSession::KeymanGetContext(int n, PWSTR buf)   // I3567
 
 BOOL CKMTipTextService::TIPNotifyActivate(GUID *guidProfile)   // I3581   // I4274
 {
-  _UninitKeystrokeSink();   // I3590
+  LogEnter();
+
+  //_UninitKeystrokeSink();   // I3590
   if(!_InitKeystrokeSink()) {
+    SendDebugMessage(L"TIPNotifyActivate: _InitKeystrokeSink failed");
     return FALSE;   // I3590
   }
 
   if(!_LoadKeyman()) {
+    DebugLastError(L"GetModuleHandle(keymanxx.dll)");
     return FALSE;
   }
 
 	PTIPACTIVATEKEYBOARDFUNC pTIPActivateKeyboard = (PTIPACTIVATEKEYBOARDFUNC) GetProcAddress(_hKeyman, "TIPActivateKeyboard");
 	if(!pTIPActivateKeyboard) { 
-    return FALSE; 
+    DebugLastError(L"GetProcAddress(TIPActivateKeyboard)");
+    return FALSE;
   }
 	if(!(*pTIPActivateKeyboard)(guidProfile)) { 
-    return FALSE; 
+    SendDebugMessage(L"TIPNotifyActivate: TIPActivateKeyboard failed");
```

Comment:
The indention looks off because there's some tabs to replace in ll 295, 296, 300